src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import classnames from 'classnames'
+
+type FooterProps = {
+  className?: string
+  big?: boolean
+  medium?: boolean
+  slim?: boolean
+  /**
+   * Content in upper footer section, e.g. navigation, searchbar, signup form
+   */
+  primary: React.ReactNode
+  /**
+   * Content in lower footer section, e.g. contact information
+   */
+  secondary: React.ReactNode
+}
+
+// TODO: Add in "Return to Top" handling
```

Comment:
`returnToTop` could be an option prop -- thoughts?  Or should this component just render `children`?  It seems like it is supposed to also encapsulate styles so that is why I made `primary` and `secondary` in the first place.

Reply:
I agree that `primary` and `secondary` is the way to go, since all of the USWDS footers divide up the content that way pretty explicitly. What if we had an optional `onReturnToTop` fn prop that, if it exists renders the Return to top link and hooks up the event handler?

Reply:
Made `returnToTop` a component when I realized that if I only made the handler a prop I am hardcoding English, and that specific phrase, into our repo. I'll make an issue about lang and i18n so we can discuss.